apps/src/code-studio/pd/application/teacher/ChooseYourProgram.jsx

```diff
@@ -45,19 +55,33 @@ export default class ChooseYourProgram extends LabeledFormComponent {
         (csHowManyMinutes * csHowManyDaysPerWeek * csHowManyWeeksPerYear) / 60;
     }
 
-    let courseNotes = null;
-    if (this.props.data.program && courseHours !== null) {
-      if (this.props.data.program.includes('Discoveries')) {
-        if (courseHours < 50) {
-          courseNotes = 'csd';
-        }
-      } else if (this.props.data.program.includes('Principles')) {
-        if (courseHours < 100) {
-          courseNotes = 'csp';
+    let belowMinCourseHours = false;
+    let program = this.props.data.program;
+    let minCourseHours = MIN_CSP_HOURS;
```

Comment:
up to you, but the first part of this could be replaced with 
```suggestion
    let minCourseHours = (program && program.includes('Discoveries')) ? MIN_CSD_HOURS : MIN_CSP_HOURS;
```